admin/assets/stylesheets/elements/_top_navigation.scss

```diff
@@ -0,0 +1,62 @@
+// Breadcrumbs
+.breadcrumbs {
+  padding-top: $gutter-one-third;
+  padding-bottom: $gutter-half;
+
+  div {
+    display: inline-block;
+    font-size: medium;
+
+    a {
+      font-size: medium;
+    }
+  }
+}
+
+.breadcrumb-separator {
+  color: $grey-2;
+  font-weight: bold;
+  margin-left: $gutter-one-third;
+  margin-right: $gutter-one-third;
+}
+
+// Back button
+.back-button-wrapper {
+  padding-top: $gutter-one-third;
+  padding-bottom: $gutter-half;
+}
+
+.back-button {
+  background: url("/images/icon-caret-left.png") left 0.2em no-repeat;
+  border-bottom: 1px solid $text-colour;
+  color: $text-colour;
+  display: inline-block;
+  font-size: 16px;
+  padding-left: 0.75em;
+  text-decoration: none;
+
+  &:link,
+  &:active,
+  &:visited,
+  &:hover {
+    color: $text-colour;
+    text-decoration: none;
+  }
+}
+
+@media (max-width: 640px) {
```

Comment:
@GeorgeChatzigiannis I believe from line 47 to 62 can be deleted as it's not being used.

Reply:
done